tools/submit.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+if [[ $# < 1 ]]
+then
+    echo "Usage: ./submit.sh JOB, where JOB is the job script."
+    exit 1
+fi
+
+ERR=0
+
+BIN=$(sed -n 's/BIN\s*=\s*//p' < $1)
+CFG=$(sed -n 's/CFG\s*=\s*//p' < $1)
+
+if ! [[ -f $BIN ]]
+then
+   echo "Binary $BIN not found!"
+   exit 1
+fi
+
+if ! [[ -f $BIN ]]
+then
+   echo "Config $CFG not found!"
+   exit 1
+fi
+
+./check_vlasiator_cfg.sh $BIN $CFG > check.txt
+
+if grep -q "Invalid options" check.txt 
+then
+    echo "Invalid options in config file, check check.txt"
+    echo ""
+    ERR=1
+fi
+
+SW=$(sed -n s/^file_..\\s*=\\s*//p < $CFG)
+
+if ! test -f $SW
+then
+    echo "Solar wind file $SW not found!"
+    echo ""
+    ERR=1
+fi
+
+MF=$(sed -n s/^atmosphericModelFile\\s*=\\s*//p < $CFG)
+if [[ -z $MF ]]
+then
+    MF=NRLMSIS.dat
```

Comment:
What if there's no ionosphere present in the run? In that case we don't want to check for this file.

Reply:
Now defaults to what `check_vlasiator_cfg.sh` gives, which is empty (always passes `test`) for non-ionosphere runs

Reply:
Sorry to nitpick, but the ionospheric boundary does default to using `NRLMSIS.dat`. So What this should do is a) check if an ionospheric boundary is contained in the config, and then either verify that a configured atmospheric model file exists, or if one isn't configured, that the default file NRLMSIS.dat exists.

Reply:
`check_vlasiator_cfg.sh` gives under unused options `ionosphere.atmosphericModelFile (=NRLMSIS.dat)` on ionosphere runs, which is parsed by the script. In non-ionosphere runs the unused option is absent and the script `sed`s an empty string for `$MF`.

Reply:
Ah, ok, good then!